src/docs/product/issues/issue-details/performance-issues/consecutive-http/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Consecutive HTTP"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/consecutive-http/
+description: "Learn more about Consecutive HTTP and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Consecutive HTTP issues are created when a set of sequential http spans are consistently exceeding a duration threshold.
+
+## Detection Criteria
+
+The detector for this performance issue looks for a set of sequential, non-overlapping http spans. It intentionally ignores non-http spans when finding this set, as the browser commonly performs other asynchronous operations such as loading resources, between http spans. 
+Once this sequence is found, the following must hold true for each HTTP span.
+
+- The http method must be a GET, POST, DELETE, PUT, PATCH
+- The http url must not begin with _next/static/ or _next/data/
+
+Once these spans are found, the following must also hold true:
+
+- The duration of each http span must exceed a threshold.
+- The time between each http span must exceed a threshold.
+- The number of sequential http spans must exceed a threshold.
+
+If Sentry isn't detecting a Consecutive http issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
```

Comment:
```suggestion
If Sentry doesn't detect a Consecutive HTTP issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
```